themes/theme-ios11/components/FilterBar/components/Content/index.jsx

```diff
@@ -21,7 +21,7 @@ function FilterBarContent() {
         <Sort />
         <FilterButton />
       </div>
-      <FilterChips filters={filters} routeId={routeId} scrollTop={scrollTop} />
+      <FilterChips filters={filters} routeId={routeId} scrollTop={scrollTop} pattern={pattern} />
```

Comment:
not sure yet if the `FilterChips` component will still contain logic related to the route pattern  when this PR is merged, but i would try to avoid adding additional props to components when you can easily access the the needed values from inside the component.

in this case you could inject the route props into `FilterChips` by wrapping it with the `withRoute` HOC.